### PR TITLE
Add Pathfinders card Wetlands.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

**Comment by kberg** [comment]
Depends on #3976

**Comment by bafolts** [comment]
Can you merge this with main? The diff on github is showing a lot of modifications that already exist in main.

**Comment by kberg** [comment]
> Can you merge this with main? The diff on github is showing a lot of modifications that already exist in main.

Done. That was because 
image

**Comment by kberg** [comment]
> > Can you merge this with main? The diff on github is showing a lot of modifications that already exist in main.
> 
> Done. That was because image

(should be good now.)

File: src/cards/base/ArcticAlgae.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class ArcticAlgae extends Card implements IProjectCard {
   }
 
   public onTilePlaced(cardOwner: Player, activePlayer: Player, space: ISpace) {
-    if (space.tile?.tileType === TileType.OCEAN) {
```
Comment (bafolts) [inline src/cards/base/ArcticAlgae.ts:37]: This seems like different logic. Is there anything that can place an ocean tile on the non base places? Did this have to change with this PR?

Comment (bafolts) [reply src/cards/base/ArcticAlgae.ts:37]: oh ok I read this wrong. The base ocean tiles is not the places you can place the tile but the tile types themselves.

**Review by bafolts — APPROVED** [review]

File: src/cards/base/ArcticAlgae.ts (hunk 1 above)
Comment (kberg) [reply src/cards/base/ArcticAlgae.ts:37]: Do you have a better name for it?

**Pull request merged**